assignments/views.py

```diff
@@ -0,0 +1,18 @@
+from assignments.models import Assignment
```

Comment:
run `isort -rc --atomic -y .` to sort your imports as specified in the `PEP8` standard